corehq/pillows/mappings/app_mapping.py

```diff
@@ -1,6 +1,6 @@
 from corehq.util.elastic import es_index
 
-APP_INDEX = es_index("hqapps_2016-01-05_1821")
+APP_INDEX = es_index("hqapps_2016-01-29_0013")
```

Comment:
I made a note of it [here](https://github.com/dimagi/commcare-hq/blob/master/corehq/doctypemigrations/README.md#commit-4-maybe-flip-pillowtop-andor-elasticsearch).  I recommend skimming that doc if you're interested in more about how this process works.